### DISCUSSION TITLE
Syntax error in Tabmenu / Router code

### DISCUSSION THREAD

**Discussion by Kmarcig91** [primefaces/community#1229] [PrimeVue] [answered]
There is an extra "{" in the Tabmenu / Router code (line 25)
https://primevue.org/tabmenu/#router

![tabmenu](https://github.com/primefaces/community/assets/16471355/ca712343-5ede-441e-baf3-c8a506b3afba)

**Comment by tugcekucukoglu** [comment, marked as answer]
Typo will be fixed.

Thank you!
